CHANGES.rst

```diff
@@ -10,7 +10,7 @@ Bugfixes:
   This prevents lots of ResourceWarnings about unclosed sockets.
   Fixes issues `636 <https://github.com/plone/plone.restapi/issues/636>`_
   and `648 <https://github.com/plone/plone.restapi/issues/648>`_.
-  [maurits]
+  [maurits] [gforcada]
```

Comment:
@gforcada we usually use a comma-separated list here, right?

Reply:
No idea, I will change it